src/pages/index.tsx

```diff
@@ -23,6 +30,8 @@ const Index: FC = () => {
     if ('tasks' in response) {
       tasks = response.tasks;
       tasks.sort((a:task, b:task) => +a.endsOn - +b.endsOn);
+      const statusOrder = [ACTIVE, ASSIGNED, COMPLETED, PENDING, UNASSIGNED];
```

Comment:
Please move this on line no: 18 as this value is known prior to execution.
Also, please make the name STATUS_ORDER.
https://javascript.info/variables#uppercase-constants

Reply:
Made the requested changes